### PR TITLE
Extrahorns

### PR DESCRIPTION
**Pull request by DonutNinja**

### CODE REVIEW

File: mods/extrahorns/init.lua [hunk 1]
```diff
@@ -0,0 +1,185 @@
+-- large horn
+minetest.register_craftitem ("extrahorns:largehorn", {
```
Comment (luk3yx) [inline mods/extrahorns/init.lua]: Why the space between `minetest.register_craftitem` and `(`?

File: mods/extrahorns/init.lua [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+			max_hear_distance = 350,
+			gain = 1,
+		})
+	end,
+})
+
+minetest.register_craft ({
+	output = "extrahorns:bagpipes",
+	recipe = {
+		{ "default:stick", "default:stick", "default:gold_ingot"} ,
```
Comment (luk3yx) [inline mods/extrahorns/init.lua]: The space between the `{` and the `"` makes it inconsistent with the right-hand side. Either add a space there or remove this space.

File: mods/extrahorns/init.lua [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+		{ "", "default:stick", ""},
+		{ "", "", "default:stick"},
+	}
+})	
+
+-- drum
+minetest.register_node("extrahorns:wardrum", {
+    description = "War Drum",
+    tiles = {
+        "extrahorns_drum_top.png", -- Top
```
Comment (luk3yx) [inline mods/extrahorns/init.lua:133]: Inconsistent use of tabs and spaces.

**Review by luk3yx — CHANGES_REQUESTED** [review]
Some minor formatting issues should be addressed.

You should also add a `LICENSE.md` specifying what license this mod is under.

**Review by ClobberXD — CHANGES_REQUESTED** [review]
This branch contains outdated changes. Please rebase it upon the latest `master`.

**Pull request merged**